pkgs/servers/certificate-transparency/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, pkgs, ...}:
+
+stdenv.mkDerivation rec {
+  name = "certificate-transparency";
+
+  version = "60bd6f9394897e634b54ecc9be220bb3e8f2e1a5";
```

Comment:
`version` is not an attribute used by `mkDerivation`. As this package doesn't have releases please provide a timestamp instead so that upgrades are possible with `nix-env`. You have to provide the version in the `name` attribute, like:

    name = "certificate-transparency-git-2015-11-18";